mautic/mautic
# Add background-image and properties to section

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | Y
| Automated tests included? | N
| Related user documentation PR URL | N
| Related developer documentation PR URL | N
| Issues addressed (#s or URLs) | N
| BC breaks? | N
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Thanks to @chocopowwwa for is work.
It's a corrected version of his PR: https://github.com/mautic/mautic/pull/5665
This add background image, background size and background repeat to section in Email builder and Landing builder.

[//]: # ( As applicable: )
#### Steps to test this PR:
1. Apply PR
2. Clear cache and regenerate assets
3. Open Email Builder or Landing Builder, Select a data-section element
4. Then you'll see additional field in Customize Section menu to change the background image, background repeat and background size. Input some value to it, then save.

![screenshot_2018-11-06 edit page - landing background mautic](https://user-images.githubusercontent.com/27768270/48077214-250b6980-e1e7-11e8-919b-7abba443f375.png)

5. Look the preview, you have now customized background-image.
